From: Raphael GILLES <rgilles@student.42.fr>
Subject: [PATCH 1/1] kernel build: Added an EXTRAVERSION string
Add the string "-thor_kernel" to the EXTRAVERSION variable in Makefile

Signed-off-by: Raphael GILLES <rgilles@student.42.fr>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 48a044bfe..f3317e95c 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 4
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION = -thor_kernel
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
-- 
2.37.2

